### PR TITLE
[HOPSWORKS-1372] Upgrade the ELK stack to 7.2 OSS and use the OpenDis…

### PR DESCRIPTION
**Pull request by maismail**
…tro Security Plugin
https://logicalclocks.atlassian.net/browse/HOPSWORKS-1372
tests: https://github.com/logicalclocks/hops-testing/pull/565

### CODE REVIEW

File: attributes/default.rb [hunk 1]
```diff
@@ -59,3 +59,14 @@
 
 default['hopslog']['private_ips']         = ['10.0.2.15']
 default['hopslog']['public_ips']          = ['10.0.2.15']
+
+# Kibana Opendistro Security plugin
+default['kibana']['opendistro_security']['url']   = "#{node['download_url']}/opendistro_security_kibana_plugin-#{node['elastic']['opendistro']['version']}.zip"
```
Comment (SirOibaf) [inline attributes/default.rb]: Please tabulate these.

File: recipes/_kibana.rb [hunk 2]
```diff
@@ -1,22 +1,19 @@
...
 
-
-elastic = private_recipe_ip("elastic", "default") + ":#{node['elastic']['port']}"
-kibana = private_recipe_ip("hopslog", "default") + ":#{node['kibana']['port']}"
-
-numRetries=10
-retryDelay=20
+elastic_url = any_elastic_url()
+elastic_addrs = all_elastic_urls_str()
+kibana_url = node['kibana']['opendistro_security']['https']['enabled'].casecmp?("true") ? "https://#{node['fqdn']}:#{node['kibana']['port']}" : "http://#{my_private_ip}:#{node['kibana']['port']}"
```
Comment (SirOibaf) [inline recipes/_kibana.rb]: Does this work if we deploy in localhost mode? Will Kibana be listening on 0.0.0.0?

File: attributes/default.rb (hunk 1 above)
Comment (maismail) [reply attributes/default.rb]: done

File: recipes/_kibana.rb (hunk 2 above)
Comment (maismail) [reply recipes/_kibana.rb]: Yes, kibana will listen on 0.0.0.0

**Pull request merged**